Sources/Workspace/WorkspaceConfiguration.swift

```diff
@@ -76,6 +76,11 @@ extension Workspace {
         public var sharedRegistriesConfigurationFile: AbsolutePath? {
             self.sharedConfigurationDirectory.map { DefaultLocations.registriesConfigurationFile(at: $0) }
         }
+        
+        // Path to temporary files related to running plugins in the workspace
+        public var pluginDataDirectory: AbsolutePath {
```

Comment:
to be consistent this should be named `pluginWorkingDirectory`

Reply:
Sounds good, that makes sense.